[2024-11-03T04:19:30.214+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T04:19:30.220+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.db_to_schema manual__2024-11-03T04:19:17.705124+00:00 [queued]>
[2024-11-03T04:19:30.223+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.db_to_schema manual__2024-11-03T04:19:17.705124+00:00 [queued]>
[2024-11-03T04:19:30.223+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-03T04:19:30.227+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): db_to_schema> on 2024-11-03 04:19:17.705124+00:00
[2024-11-03T04:19:30.230+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=198) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T04:19:30.231+0000] {standard_task_runner.py:72} INFO - Started process 200 to run task
[2024-11-03T04:19:30.232+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'db_to_schema', 'manual__2024-11-03T04:19:17.705124+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpqv38g4fn']
[2024-11-03T04:19:30.233+0000] {standard_task_runner.py:105} INFO - Job 295: Subtask db_to_schema
[2024-11-03T04:19:30.250+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.db_to_schema manual__2024-11-03T04:19:17.705124+00:00 [running]> on host f35298df57b7
[2024-11-03T04:19:30.276+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='db_to_schema' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T04:19:17.705124+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T04:19:17.705124+00:00'
[2024-11-03T04:19:30.276+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T04:19:31.614+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-03T04:19:31.635+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T04:19:31.636+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=db_to_schema, run_id=manual__2024-11-03T04:19:17.705124+00:00, execution_date=20241103T041917, start_date=20241103T041930, end_date=20241103T041931
[2024-11-03T04:19:31.673+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T04:19:31.688+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-03T04:19:31.689+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
